horace_core/sqw/file_io/@faccess_sqw_v3_2/private/upgrade_file_format_.m

```diff
@@ -19,7 +19,8 @@
 end
 [~,acc] = fopen(obj.file_id_);
 if ~ismember(acc,{'wb+','rb+'})
-    clear new_obj.file_closer_;  % as file is closed on output of reopen to write
+    new_obj.file_closer_.delete();  % as file is closed on output of reopen to write
+    new_obj.file_closer_ = [];  % as file is closed on output of reopen to write
     new_obj = new_obj.fclose();  % in case the previous does not work, and if it does, makes no harm
```

Comment:
This seems like concerning overkill.

Reply:
@abuts Please reference this ticket in comments in the code so we know why you're doing it

Reply:
@cmarooney-stfc I have done this but this is monkey job. The ticket would probably involve more work than this, so it just typing obvious in random places.